src/test/FuzzerImpl.cpp

```diff
@@ -841,11 +1048,21 @@ TransactionFuzzer::initialize()
             }
         }
 
-        attemptToApplyOps(ltx, mSourceAccountID, ops.begin(), ops.end(), *mApp);
+        attemptToApplyOps(ltx, mSourceAccountID, ops.begin(), ops.end(), *mApp,
+                          true, false);
 
         ltx.commit();
     }
 
+    // Cache the source account's sequence number for future calls to
+    // ->inject().
+    mPostInitSourceSeqNum = getSequenceNumber(ltxOuter, mSourceAccountID);
+
+    // Remember the current stored ledger keys so that we can reset to the
+    // current (post-initialization) state before each ->inject().
+    std::copy(mStoredLedgerKeys.begin(), mStoredLedgerKeys.end(),
```

Comment:
What I think I'm doing here is generating `mPostInitLedgerKeys` first in the constructor, and then later in the constructor copying that over `mStoredLedgerKeys` by calling `resetStoredLedgerKeys()`.  So after the constructor, both arrays contain the same (autocheck-generated) elements.  Setup then overwrites some of `mStoredInitLedgerKeys`.  At the end of setup we copy that array back to `mPostInitLedgerKeys`, which is immutable thereafter.  At the beginning of each test cycle we restore `mStoredInitLedgerKeys` from `mPostInitLedgerKeys`.

I'm copying the whole array because now that we're filling it upon construction and then indexing into it directly with a byte rather than keeping a counter, I wouldn't be able to restore only entries known to have been modified without tracking those modifications in a bitmap or something.

Am I messing that up?  (Even if it's right, I need to make it much more clearly so by separating this mechanism into its own commit as part of breaking up the big one.)

Reply:
(Though, as you point out below, just passing in a ledger-key array by itself to the unpacker obviates the need for save/restore entirely, so the above code can simply be removed!)

Reply:
(The code is gone now.)